scripts/server.js

```diff
@@ -4,12 +4,18 @@ import gaze from 'gaze/lib/gaze.js';
 import StaticServer from 'static-server/server.js';
 
 import { buildCSS } from './build_css.js';
+// import { buildData } from './build_data.js';
 
 
 gaze(['css/**/*.css'], (err, watcher) => {
   watcher.on('all', () => buildCSS());
 });
 
+// // added gaze watcher for all .js files
+// gaze(['**/*.js'], (err, watcher) => {
+//   watcher.on('all', () => buildData());
+// });
+
```

Comment:
I am still not knowledgeable enough about the code base to review the contents of your code, but my only comment would be to remove some of the leftover comments, such as these, on scripts/server.js :)

Great work, Lauren!

Reply:
Great callout @voscarmv - often code reviews are great for catching things like this! Though, in this case the PR is a 'draft' PR that Lauren wanted to push up so I could test it- I expect that further refactors will be required, and at this point it's still okay to have extra comments/other things flying around. 

If this were a non-draft PR, yeah, extra comments, console.logs, etc. would need to be stricken. :)